src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -1152,7 +1156,7 @@ subroutine init_atm_static(mesh, dims, configs)
     call mpas_f_to_c_string(fname, c_fname)
 
     call read_geogrid(c_fname,rarray_ptr,nx,ny,nz,isigned,endian, &
-                      scalefactor,wordsize,istatus)
+                      wordsize,istatus)
     call init_atm_check_read_error(istatus,fname)
     vegfra(-2:1250,-2:1253,1:12) = rarray(1:1253,1:1256,1:12)
```

Comment:
Although the scaling factor is 1.0, should we scale `rarray` by `scalefactor` here (and a few other places, below) for consistency?

Reply:
Yes, that is probably the best idea. Should we also update the terrain, landuse and soil category to also be scaled by their scale factor? If so I'll think we'll need to do so after computing the average height for each cell (or most dominate category). I.E.:
``` Fortran
 do iCell = 1, nCells
    ter(iCell) = real(ter_integer(iCell), kind=R8KIND) / real(nhs(iCell), kind=R8KIND)
    ter(iCell) = ter(iCell) * scalefactor
 end do
```

Reply:
I think we should definitely update the terrain field -- in principle, if the terrain dataset did have a non-unity scaling factor (e.g., a scale factor of 0.001, so that we had vertical resolution of 1 mm), we wouldn't get correct results without scaling.

I'm unsure that scaling would ever be used for categorical fields, so I'm fine with not performing any scaling for land use category and soil category. Categories are necessarily integer values, and it's hard to imagine a dataset in which, e.g., someone provided categories 100, 110, 120, 130, etc. but represented them in a file with values 10, 11, 12, 13 and a scaling factor of 10.0.

Reply:
That makes sense! Okay, I just pushed an update with the above request.

Reply:
I just scanned through the diffs so I may be mistaken, but are we still missing code to scale `rarray` before lines 1177 and 1368?

Reply:
You were correct. Apologies about missing that one. I have just pushed an update. We should now be scaling `rarray` after any call to `read_geogrid`.

Reply:
It looks like we're still missing code to scale `rarray` around L.1368.

Reply:
Okay, scaling has been added to rarray around line 1368.